test/Test/Data/List.purs

```diff
@@ -398,6 +407,27 @@ testList = do
   log "append should be stack-safe"
   void $ pure $ xs <> xs
 
+  log "mapReverse should work the same as slower naive version"
+  --quickCheck mapReverseLaw
+
+  log "naiveAddIndexReverse should work the same as slower naive version"
+  --quickCheck naiveAddIndexReverse
```

Comment:
I'm having trouble getting `quickcheck` to work as a bower dev dependency. I'll show you want I mean in the next commit. Maybe CI will show the same error.